### PR TITLE
Goto page

### PR DESCRIPTION
**Pull request by ChesterLY**
- New version of the Goto_Page (clicking the direction button)
- Fully integrated including callback functions and path finding works
- Coordinate.dart only has toString changed from displaying LatLng to the name
- Old goto page file still exist at the moment as a backup for the feature
- Snackbar for notification is not working, related to bug issue #87

### CODE REVIEW

**Comment by ChesterLY** [comment]
I will be continuing to work on this feature, the unit and widget testing will be part of the next PR.

File: lib/views/goto_page.dart [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                          : widget.confirmDirection(new List<Coordinate>.from(
+                              [_searchedStart, _searchedDestination])),
+                      Navigator.pop(context)
+                    };
+            }),
+      ),
+    );
+  }
+
+  //The snackbar is current not working but is a QoL implementation, will be added as a bug
```
Comment (ooms97) [inline lib/views/goto_page.dart]: It might be a better idea to just remove this code completely if it's just an added feature.

Comment (ChesterLY) [reply lib/views/goto_page.dart]: Gotcha, it is removed on the latest commit

**Comment by ooms97** [comment]
Thanks for the changes, looks good now!

**Pull request merged**